### DISCUSSION TITLE
Using an Analyst VM with the SO AWS AMI

### DISCUSSION THREAD

**Discussion by rmarkscott** [Security-Onion-Solutions/securityonion#3563] [Unsupported Versions] [answered]
I want to use a local Analyst VM to access SO 2.3 running on AWS (the official SO AMI). I am unsure how to make this connection happen. Will it use http or ssh to connect? Anyone done this? Any guidance?

**Comment by jertel** [comment, marked as answer]
If you're using the Analyst VM to connect to the SOC web interface then it will need HTTPS access to the AWS EC2 instance.  You can also allow SSH if you plan to connect to the grid terminal for some reason. Also, make sure you add the Analyst VM's CIDR to the `so-allow` command.

**Comment by rmarkscott** [comment]
Thank you, Jason. 

Mark Scott, M.S., Assistant Professor, GCIA
Cybersecurity Program Coordinator
Member - The FBI’s InfraGard Program
Freed-Hardeman University | 158 E. Main Street | Henderson, TN 38340
731.989.6779

http://www.fhu.edu/cybersecurity
